src/functions-reference/unbounded_continuous_distributions.Rmd

```diff
@@ -176,28 +176,73 @@ dropping constant additive terms.
 
 ### Stan Functions
 
+<!-- real; normal_id_glm_lpdf; (real y | matrix x, real alpha, vector beta, real sigma); -->
+\index{{\tt \bfseries normal\_id\_glm\_lpdf }!{\tt (real y \textbar\ matrix x, real alpha, vector beta, real sigma): real}|hyperpage}
+
+`real` **`normal_id_glm_lpdf`**`(real y | matrix x, real alpha, vector beta, real sigma)`<br>\newline
+The log normal probability density of y given location `alpha + x * beta`
+and scale `sigma`, where a constant intercept `alpha`, `sigma` and dependent variable value `y` are
+used for all observations. The number of columns of `x` needs to match
+the length of the weight vector `beta`.
+If `x` and `y` are data (not parameters) this function can be executed on a GPU.
+
+<!-- real; normal_id_glm_lpdf; (real y | matrix x, vector alpha, vector beta, real sigma); -->
+\index{{\tt \bfseries normal\_id\_glm\_lpdf }!{\tt (real y \textbar\ matrix x, vector alpha, vector beta, real sigma): real}|hyperpage}
+
+`real` **`normal_id_glm_lpdf`**`(real y | matrix x, vector alpha, vector beta, real sigma)`<br>\newline
+The log normal probability density of y given location `alpha + x * beta`
+and scale `sigma`, where a constant `sigma` and dependent variable value`y` are used for all
+observations and an intercept `alpha` is used that is allowed to vary
+with the different observations. The number of rows of the independent
```

Comment:
[optional]
with the different observations -> by observation

Reply:
fixed